exonum-java-binding/testkit/src/test/java/com/exonum/binding/testkit/TestKitTest.java

```diff
@@ -70,6 +72,7 @@
   private static final ZonedDateTime TIME =
       ZonedDateTime.of(2000, 1, 1, 1, 1, 1, 1, ZoneOffset.UTC);
 
+  @Disabled
```

Comment:
Same here (at class level).

Reply:
There is one test `createTestKitWithBuilderForMultipleSameServices` from class that passes 🙃